package.json

```diff
@@ -53,5 +26,32 @@
     "react-draggable": "2.2.1",
     "react-tooltip": "3.0.13",
     "watchify": "3.7.0"
-  }
+  },
+  "engines": {
+    "firefox": ">=38.0a1"
+  },
+  "homepage": "https://github.com/meandavejustice/min-vid",
+  "license": "MPL-2.0",
+  "main": "index.js",
+  "permissions": {
+    "multiprocess": true
+  },
+  "precommit": "package",
```

Comment:
Sorry for the noise, I think I ran `fixpack` which sorts everything into a rough a-Z order so things got shuffled, but this is the only line that changed:

```diff
diff --git a/package.json b/package.json
index 71a1f24..0977769 100644
--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "dev": "npm run watch-script & npm run watch & http-server -c-1",
     "deploy": "npm run package && ./bin/sign"
   },
-  "precommit": "build-script, package",
+  "precommit": "package",
   "license": "MPL-2.0",
   "dependencies": {
     "get-video-id": "1.0.0"
```